parm/evs_config/cam/config.evs.cam_firewxnest.prod

```diff
@@ -26,7 +26,7 @@ export RUN_GRID2OBS_PLOTS="NO"
 #model_list:             model names
 #model_stat_dir_list:    directory path to model .stat files
 #model_file_format_list: file format of model files
-export model_list="NAM_FIREWXNEST"
+export model_list="NAM_FIREWXNEST, RRFS_FIREWXNEST"
```

Comment:
Similar question as before, can you provide context as to why the MODELNAME is changed in the high level scripts to remove specific models, then added in other areas of the code?

Reply:
Here, plots was expanded to include rrfs_firewxnest so that graphics include both nam_firewxnest and rrfs_firewxnest lines.  Thus, "MODELNAME" was broadened from "nam_firewxnest" to "firewxnest", and then the list of models to plot was expanded to include rrfs_firewxnest.